src/mgr/DaemonServer.cc

```diff
@@ -18,6 +18,7 @@
 #include "include/str_list.h"
 #include "auth/RotatingKeyRing.h"
 #include "json_spirit/json_spirit_writer.h"
+#include <boost/algorithm/string.hpp>
```

Comment:
nit, please follow https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes and put this line right after `#include "DaemonServer.h"`.

Reply:
Thanks for your patience.